website/content/en/highlights/2022-08-16-0-24-0-upgrade-guide.md

```diff
@@ -17,9 +17,11 @@ Vector's 0.24.0 release includes **breaking changes**:
 4. [`ndjson` on sink `encoding` is now `json` encoding + `newline_delimited` framing](#sink-encoding-json)
 5. [VRL type definition updates](#vrl-type-def)
 
+
 and **deprecations**:
 
 1. [`internal_metrics` source cardinality metric change]
+2. [Imminent removal of deprecated names for a select set of sources and sinks](#deprecated-components)
```

Comment:
Noticed the link name here shows "sources and sinks" whereas the heading below contains "sources, transforms, and sinks" , was that intentional ?

Reply:
Woops, good eye: not intentional.

I'll get that fixed.